Code/GraphMol/MolOps.cpp

```diff
@@ -1058,7 +1058,9 @@ void addHapticBond(RWMol &mol, unsigned int metalIdx,
   auto dummyAt = new QueryAtom(0);
   dummyAt->setQuery(makeAtomNullQuery());
 
-  unsigned int dummyIdx = mol.addAtom(dummyAt);
+  bool updateLabel = true;
+  bool takeOwnwership = true;
+  unsigned int dummyIdx = mol.addAtom(dummyAt, updateLabel, takeOwnwership);
```

Comment:
wow, I'm surprised this leak hasn't been noticed before.
Bad code review on my part when this was merged

Reply:
Yeah, I was surprised too

Reply:
Bad code on my part in the first place :-(.  It's quite new code, only in the current release I think, so not so much chance for it to show up.
Much as I like developing on a Mac, I really do miss valgrind.

Reply:
I haven’t. I will take a look, thanks.